### DISCUSSION TITLE
Cannot star empty repository

### DISCUSSION THREAD

**Discussion by cipri-tom** [community/community#65200] [Repositories]
### Select Topic Area

Bug

### Body

Some scientific papers announce they will release code at a certain repository, but for now is still empty (common practice to not release the code until the paper is peer reviwed).

In the case of an empty repository, we cannot star or watch it, so that we'd be updated of when the code is released.

I believe this is a UI bug.

![Screenshot 2023-08-28 at 11 16 49](https://github.com/community/community/assets/2991890/d85459d5-9149-4269-b012-a1bcd9de4f5a)

**Comment by Sahil002620Q** [comment]
I think this could be confusing for users because starring is normally used to bookmark repositories regardless of whether they currently contain code. Allowing users to star an empty repository could still be useful, especially when someone wants to follow a project from its beginning.
